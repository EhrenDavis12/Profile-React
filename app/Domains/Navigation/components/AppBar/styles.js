import styled from 'styled-components';

export const Nav = styled.div`
  width: 100%;
  padding: 20px;
  background-color: #00afcc;
  color: #fff;
  font-size: 24px;
  display: flex;
  justify-content: space-between;
  opacity: 0.9;
  filter: alpha(opacity=90);
`;
