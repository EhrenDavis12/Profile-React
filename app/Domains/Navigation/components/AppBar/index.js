/**
 *
 * AppBar
 *
 */

import React from 'react';
import PropTypes from 'prop-types';
// import styled from 'styled-components';
import './styles.css';
import FontAwesome from 'react-fontawesome';
import BasicButton from 'Domains/SharedKernel/components/Buttons/BasicButton';

function AppBar({ toggleDrawer }) {
  // return <div className={styles.appBar}>coder daily</div>;
  return (
    <div className="appBar">
      <BasicButton className="iconButton" onClick={toggleDrawer}>
        <FontAwesome className="icon" name="bars" />
      </BasicButton>
      <div className="heading">Coder Daily</div>
      <div className="linkContainer">Log In</div>
    </div>
  );
}

AppBar.propTypes = {
  toggleDrawer: PropTypes.func.isRequired,
};

export default AppBar;
