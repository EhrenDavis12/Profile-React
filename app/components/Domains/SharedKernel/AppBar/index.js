/**
 *
 * AppBar
 *
 */

import React from 'react';
import PropTypes from 'prop-types';
// import styled from 'styled-components';
import './styles.css';
import FontAwesome from 'react-fontawesome';
import BasicButton from 'components/Domains/SharedKernel/Buttons/BasicButton';

function AppBar({ toggleDrawer }) {
  // return <div className={styles.appBar}>coder daily</div>;
  return (
    <div className="appBar">
      <BasicButton className="iconButton" func={toggleDrawer}>
        <FontAwesome className="icon" name="bars" />
      </BasicButton>
      {/* <div className="iconButton">
        <FontAwesome className="icon" name="bars" />
      </div> */}
      <div className="heading">Coder Daily</div>
      <div className="linkContainer">Log In</div>
    </div>
  );
}

AppBar.propTypes = {
  toggleDrawer: PropTypes.func.isRequired,
};

export default AppBar;
