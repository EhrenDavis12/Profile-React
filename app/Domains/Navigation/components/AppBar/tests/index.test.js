import React from 'react';
import renderer from 'react-test-renderer';
// import { mount } from 'enzyme';
// import { enzymeFind } from 'styled-components/test-utils';

import AppBar from '../index';

describe('<AppBar />', () => {
  it('renders as expected', () => {
    const tree = renderer.create(<AppBar toggleDrawer={() => {}} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
