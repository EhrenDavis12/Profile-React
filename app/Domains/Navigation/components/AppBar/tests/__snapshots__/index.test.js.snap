// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppBar /> renders as expected 1`] = `
<div
  className="styles__Nav-f0nxkg-0 gGbmCv"
>
  <div
    className="iconButton"
    onClick={[Function]}
    onKeyDown={[Function]}
    role="button"
    tabIndex={0}
  >
    <span
      aria-hidden={true}
      className="fa fa-bars icon"
    />
  </div>
  <div
    className="heading"
  >
    Ehren Davis
  </div>
  <div
    className="linkContainer"
  >
    Log In
  </div>
</div>
`;
